Skype/WebSDK/docs/PTVideoGroup.md

```diff
@@ -6,83 +6,113 @@
 
 ## Starting a group video conversation
 
-The application object exposes a conversationsManager object which we can use to create new group conversation by calling createConvresation().  After creation of the conversation object, it is helpful to setup a few event listeners for when we are connected to video, added participants, when participants are connected to video, and when we disconnect from the conversation.
-
-When either the selfParticipant or other persons are conencted to video we also need to setup the DOM element where the video should be displayed.  This configuration can be handled by getting access to the sink object by walking from the person object to the video modality to the channels collection choosing the first, or channels(0), which gives us access to the stream object which has a source object which finally points us to the sink object.  The sink object has a format property that can accept video formatting options such as Stretch, Fit, and Crop.  The sink object also exposes a container property where we can provide a DOM element where the video will be inserted.
-
-We can add participants to the conversation by calling add(...) providing a SIP URI on the participants collection of the conversation object.  We can use the videoService on the conversation object and call start() to initate the call.
-
-After the conversation and video modality are established we can begin communicating with the remote parties.  When finished click the end button to terminate the conversation.
+1. As a first step we need to create a conversation
 
+    ```js
+        var conversation = application.conversationsManager.createConversation();
+    ```
 
-### Start a group video conversation
+1. Now we are adding a listener to display our own video when the camera is turned on.
+When the listener is triggered, we can assign a DOM node as the container for the video.
 
-1. Start group video conversation, and track participant events 
+    ```js
+        conversation.selfParticipant.video.state.when('Connected', function () {
+            // video is availabe ... lets assign a container.
+            conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch'); // formats include: 'Stretch', 'Fit' and 'Crop'
+            conversation.selfParticipant.video.channels(0).stream.source.sink.container(/* DOM node such as a DIV */);
+            // the video will be rendered automatically
+        });
+    ```
+1. To render the video of the other participants in the conversation we add another listener that notifies us when
+another participant joins the conversation.
+
+    > [!Warning]
```

Comment:
I'd make this "IMPORTANT," - I think we should save WARNINGS for things like making sure not to post decrypted HTTPS traffic that might contain passwords on stackoverflow, mostly because the "warning" is pretty aggressive-looking in MSDN and we don't want to overuse it